src/jit/lir.h

```diff
@@ -204,6 +258,24 @@ class LIR final
     {
         return Range(container->FirstLIRNodeSlot(), container->LastLIRNodeSlot());
     }
+
+    //------------------------------------------------------------------------
+    // LIR::SetTreeSeq: Given a newly created, unsequenced HIR tree, sequence
+    // the tree (set gtNext/gtPrev pointers), and return a Range representing
+    // the list of nodes. It is expected this will later be spliced into the
+    // LIR graph afterwards.
+    //
+    // Arguments:
+    //    compiler - The Compiler context.
+    //    tree - The tree to sequence.
+    //
+    // Return Value: The newly constructed range.
+    //
+    static Range SetTreeSeq(Compiler* compiler, GenTree* tree)
```

Comment:
Yes. I could have sworn I'd removed it, but I've done so many rounds of rebasing and subsequent and surprising conflict resolution I wouldn't be surprised if it came back.